js/index.js

```diff
@@ -0,0 +1,333 @@
+          </ul>
+
+          <div class="action">
+            <button class="normal apper" type="button">See Project</button>
+          </div>
+        </div>
+      </div>`;
+const thirdCard = `
+      <div class="card">
+        <div class="photo">
+          <img
+            class="work-img"
+            src=""
+            alt="card-img"
+          />
+        </div>
+        <div class="left-block">
+          <div class="primary-text">
+            <h2 class="project-title"></h2>
+            <div class="frame-2">
+              <div class="Client">
+                <h2 class="CANPOY">CANPOY</h2>
+                <i class="counter fa-solid fa-circle"></i>
+                <div class="Role"><h2 class="back-end">Back End Dev</h2></div>
+                <i class="counter fa-solid fa-circle"></i>
+                <div class="Role"><h2 class="back-end">2022</h2></div>
+              </div>
+            </div>
+          </div>
+          <div class="information">
+            <p class="des-text">
+              
+            </p>
+          </div>
+
+          <ul class="tags">
+            <li class="tag"><a class="html" href=""></a></li>
+            <li class="tag"><a class="html" href=""></a></li>
+            <li class="tagj"><a class="html" href=""></a></li>
+          </ul>
+
+          <div class="action">
+            <button class="normal apper" type="button">See Project</button>
+          </div>
+        </div>
+      </div>`;
+const forthCard = `
+      <div class="card">
+        <div class="photo">
+          <img
+            class="work-img work-img1"
+            src=""
+            alt="card-img"
+          />
+        </div>
+        <div class="left-block left-block1">
+          <div class="primary-text">
+            <h2 class="project-title"></h2>
+            <div class="frame-2">
+              <div class="Client">
+                <h2 class="CANPOY">CANPOY</h2>
+                <i class="counter fa-solid fa-circle"></i>
+                <div class="Role"><h2 class="back-end">Back End Dev</h2></div>
+                <i class="counter fa-solid fa-circle"></i>
+                <div class="Role"><h2 class="back-end">2022</h2></div>
+              </div>
+            </div>
+          </div>
+          <div class="information">
+            <p class="des-text">
+              
+            </p>
+          </div>
+
+          <ul class="tags">
+            <li class="tag"><a class="html" href=""></a></li>
+            <li class="tag"><a class="html" href=""></a></li>
+            <li class="tagj"><a class="html" href=""></a></li>
+          </ul>
+
+          <div class="action">
+            <button class="normal apper" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+`;
+
+// popupcard function
+
+// for (let j = 0; j < list.length; j++){
+//   list[j].addEventListener("click", function () {
+//     popCard.classList.toggle("active");
+//   })
+// }
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly remove this commented unused code block, they make your code base not clean and professional.

Reply:
done